build/cloudbuild-automation/README.md

```diff
@@ -0,0 +1,67 @@
+### Building with CloudBuild - Automated Builds
```

Comment:
```suggestion
# Automated Builds with Cloud Build
```

Reply:
can drop other headings to lower levels too

Reply:
The other Readme(s) in the build directory have a top level heading size of H2 (##). Should we keep consistent with that - dropping all headings by 1 level instead of 2 ?